schema/descriptor.schema.json

```diff
@@ -85,7 +91,7 @@
             "id": "http://github.com/boutiques/boutiques-schema/schema-version",
             "type": "string",
             "description": "Version of the schema used.",
-            "enum": ["0.2", "0.3"]
+            "enum": ["0.4"]
```

Comment:
So this raises an interesting question - if a system wants backwards compatibility with multiple boutique versions, would the approach have to be first, take each released json schema to see which (if any) match, and go from there?

My intuition says "yes"; curious what you think :smile: